build/requirements.txt

```diff
@@ -30,6 +31,8 @@ nltk==3.9.1
 nmslib==2.1.1
 plotly==5.24.1
 plotnine==0.13.5
+openai-clip
```

Comment:
is this used for ch15 / multimodal search?

Reply:
Yes, specifically for it's multi modal encoding capabilities